easybuild/easyconfigs/s/Solids4foam/Solids4foam-2.1-foss-2023a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Binary'
+
+name = 'Solids4foam'
+version = '2.1'
+
+homepage = 'https://www.solids4foam.com/'
+description = 'A toolbox for performing solid mechanics and fluid-solid interactions in OpenFOAM.'
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/solids4foam/solids4foam/archive/']
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}]
+checksums = ['354dad483f8b086d64bf294829cf6fdf1e5784fd615578acff753791f2f391ca']
+
+builddependencies = [('Eigen', '3.4.0')]
+dependencies = [
+    ('OpenFOAM', 'v2312'),
+    ('gnuplot', '5.4.8'),
+    ('M4', '1.4.19'),
+]
+
+extract_sources = True
+
+install_cmd = 'source $FOAM_BASH && '
+# variables of target install directory
+install_cmd += 'export WM_PROJECT_USER_DIR=%(installdir)s && '
+install_cmd += 'export FOAM_USER_APPBIN=%(installdir)s/bin && '
+install_cmd += 'export FOAM_USER_LIBBIN=%(installdir)s/lib && '
+install_cmd += 'export LD_LIBRARY_PATH=%(installdir)s/lib:$LD_LIBRARY_PATH && '
+install_cmd += 'export EIGEN_DIR=$EBROOTEIGEN && '
+install_cmd += 'export S4F_NO_FILE_FIXES=1 && ./Allwmake -j %(parallel)s'
+# copy tests
+install_cmd += ' && cp -av tutorials %(installdir)s'
```

Comment:
@pavelToman This tells me that we could actually be using `MakeCp` here, and customize the build command via `build_cmd`:

```python
easyblock = 'MakeCp'

...
prebuildopts = "source $FOAM_BASH && "
...
build_cmd = './Allwmake'

files_to_copy = [...]
```

That way, the `-j %(parallel)s` is not required (will be added automatically I think), and the `extract_sources = True` can be dropped too.

Reply:
I overlooked that the `Allwmake` already installs stuff in the installation directory that is specified via `$WM_PROJECT_USER_DIR` & co, so with that in mind `MakeCp` is even worse, so don't use that :)

Add a comment abovevthe `install_cmd` though to highlight that the `Allwmake` command doesn't only build stuff, but also installs stuff, and then we can proceed like this as is